common/config/babel/.babelrc.js

```diff
@@ -70,7 +70,9 @@ process.env['COMMUNICATION_REACT_FLAVOR'] !== 'beta' &&
       inProgressFeatures: [
         // Demo feature. Used in live-documentation of conditional compilation.
         // Do not use in production code.
-        'in-progress-beta-feature-demo'
+        'in-progress-beta-feature-demo',
```

Comment:
How does this work if it is in both sections?

Reply:
The point here is you will have to add them into both sections - it's a beta feature, and it's a in progress feature